examples/classifier.c

```diff
@@ -268,11 +268,11 @@ void train_classifier_valid(char *datacfg, char *cfgfile, char *weightfile, int
     float** y_score_train = new_mat(max_train, classes, sizeof(float));
     float** y_score_valid = new_mat(max_valid, classes, sizeof(float));
 
-    int* y_true_train = malloc(max_train * sizeof(int));
-    int* y_true_valid = malloc(max_valid * sizeof(int));
+    int* y_true_train = calloc(max_train, sizeof(int));
+    int* y_true_valid = calloc(max_valid, sizeof(int));
 
-    float *train_accs = malloc((max_eval_epochs + 1) * sizeof(float));
-    float *valid_accs = malloc((max_eval_epochs + 1) * sizeof(float));
+    float *train_accs = calloc((max_eval_epochs + 1), sizeof(float));
```

Comment:
I don't think this is fixing the actual problem, but just masking it instead. Please don't merge this for now...